README.md

```diff
@@ -82,6 +82,23 @@ Network Policy agent can operate in either IPv4 or IPv6 mode. Setting this flag
 
 **Note:** VPC CNI by default creates an egress only IPv4 interface for IPv6 pods and this network interface will not be secured by the Network policy feature. Network policies will only be enforced on the Pod's primary interface (i.e.,) `eth0`. If you want to block the egress IPv4 access, please disable the interface creation via [ENABLE_V4_EGRESS](https://github.com/aws/amazon-vpc-cni-k8s#enable_v4_egress-v1151) flag in VPC CNI. 
 
+#### `conntrack-cache-cleanup-period` (from v1.0.7+)
+
+Type: Integer
+
+Default: 300
+
+Network Policy agent maintains a local conntrack cache. This configuration (in seconds) will determine how fast the local conntrack cache should be cleaned up from stale/expired entries. Based on the time interval set, network policy agent checks every entry in the local conntrack cache with kernel conntrack table and determine if the entry has to be deleted.
+
+#### `conntrack-table-cache-size` (from v1.1.3+)
```

Comment:
@jayanthvn There's typo.

It should be `--conntrack-cache-table-size`, not `--conntrack-table-cache-size`.

- **Reference**: https://github.com/aws/aws-network-policy-agent/pull/280/files#diff-62f7206b7bbef4728f451c0c5ce10690803f639d731ebf6f4364856645c7b58eR45

Reply:
Thanks, will fix this up

Reply:
@jayanthvn It's okay. I already fixed it #287